Simplified/NYPLBook.h

```diff
@@ -19,6 +14,7 @@ typedef NS_ENUM(NSInteger, NYPLBookAvailabilityStatus) {
 
 @property (nonatomic, readonly) NYPLBookAcquisition *acquisition;
 @property (nonatomic, readonly) NSString *authors;
+@property (nonatomic, readonly) NSArray *authorLinks;
```

Comment:
Okay I have a basic NYPLBookAuthor class that holds two properties for "name" and "link". I think I'm on the same page as you here.